server/metrics/global_status.go

```diff
@@ -161,7 +161,10 @@ func SetMetadataOperationInContext(ctx context.Context) {
 func AddSearchBytesInContext(ctx context.Context, value int64) {
 	reqStatus, exists := RequestStatusFromContext(ctx)
 	if reqStatus != nil && exists {
-		reqStatus.writeBytes += value
+		reqStatus.searchWriteBytes += value
+	}
+	if config.DefaultConfig.GlobalStatus.DebugMessages {
+		log.Debug().Int64("search bytes written", value).Int64("Total search bytes written", reqStatus.searchWriteBytes).Msg("Added written search bytes")
```

Comment:
Adding a `component` tag for all logs (or to begin with new logs) will help in filtering. For example if you want to pull search_usage related logs.

Reply:
I'll let @pboros fix it as he has more context on all the usage logs.